### PR TITLE
Codap, again

### PR DESCRIPTION
**Pull request by retabak**
bar-charts ready for you to take a look!

I made a new graphic organizer worksheet, largely because kids won't use contracts to make displays - and this seems like a useful reference page. Curious what you think.

### CODE REVIEW

File: lessons/codap-bar-charts/langs/en-us/pages/notes-plotting-and-displaying-data.adoc [hunk 1]
```diff
@@ -15,7 +15,7 @@
 
 @vspace{1ex}
 
-Data Scientists use *displays* to visualize data. You've probably seen some of these charts, graphs and plots yourselves! When it comes to displaying *Categorical Data*, we often rely on *bar charts* and *pie charts*. CODAP, however, only has the capacity to create *bar charts*.
+Data Scientists use *displays* to visualize data. You've probably seen some of these charts, graphs and plots yourselves! When it comes to displaying @vocab{Categorical Data}, we often rely on *dot plots* and @vocab{bar charts}. (Pie charts display categorical data, too, but CODAP doesn't have the abilitly to create them.)
```
Comment (schanzer) [inline lessons/codap-bar-charts/langs/en-us/pages/notes-plotting-and-displaying-data.adoc:18]: You may want to add a note for the teacher about why some people think Pie Charts are bad by default: https://www.data-to-viz.com/caveat/pie.html

It's not that CODAP lacks the ability to create Pie Charts -- it's that the CODAP team opposes them on principle

File: lessons/codap-bar-charts/langs/en-us/pages/plot-practice-2.adoc [hunk 2]
```diff
@@ -8,29 +8,29 @@ p { margin: 0 !important; }
...
 @n A histogram of the number of `pounds` that animals weigh.
-[cols="^1a,^1a,^1a", options="header"]
+[cols="1a,1a,1a", options="header"]
 |===
-|*Column / Attribute*
-|*Type of Data*
-|*Configuration*
+|@span{.center}{*Column / Attribute*}
+|@span{.center}{*Type of Data*}
+|@span{.center}{*Configuration*}
```
Comment (schanzer) [inline lessons/codap-bar-charts/langs/en-us/pages/plot-practice-2.adoc:20]: NO. MORE. 

@center{...} please! We made this decision almost a year ago, darnit!!!

File: lessons/codap-bar-charts/langs/en-us/pages/plot-practice-3.adoc [hunk 3]
```diff
@@ -8,17 +8,17 @@ p { margin: 0 !important; }
...
 @n A scatter-plot, using the animals `species` as the labels, `age` as the x-axis, and `weeks` as the y-axis, for all the animals from the shelter.
-[cols="^1a,^1a,^1a", options="header"]
+[cols="1a,1a,1a", options="header"]
 |===
-|*Column / Attribute*
-|*Type of Data*
-|*Measure*
+|@span{.center}{*Column / Attribute*}
+|@span{.center}{*Type of Data*}
+|@span{.center}{*Measure*}
```
Comment (schanzer) [inline lessons/codap-bar-charts/langs/en-us/pages/plot-practice-3.adoc:21]: 🤮

File: lessons/codap-bar-charts/langs/en-us/pages/plot-practice-3.adoc [hunk 4]
```diff
@@ -85,11 +85,11 @@ The display shows us the relationship between animals' age and weight.
...
 @n (Challenge) A least squares line, using the animals `species` as the labels, `pounds` as the x-axis, and `weeks` as the y-axis, for all the animals from the shelter.
-[cols="^1a,^1a,^1a", options="header"]
+[cols="1a,1a,1a", options="header"]
 |===
-|*Column / Attribute*
-|*Type of Data*
-|*Measure*
+|@span{.center}{*Column / Attribute*}
+|@span{.center}{*Type of Data*}
+|@span{.center}{*Measure*}
```
Comment (schanzer) [inline lessons/codap-bar-charts/langs/en-us/pages/plot-practice-3.adoc:92]: 🤮

File: lessons/codap-bar-charts/langs/en-us/pages/plot-practice.adoc [hunk 5]
```diff
@@ -8,29 +8,29 @@ p { margin: 0 !important; }
...
-[.FillVerticalSpace, cols="^1a,^1a,^1a", options="header"]
+@n A dot plot showing the `sex` of animals from the shelter.
+[.FillVerticalSpace, cols="1a,1a,1a", options="header"]
 |===
-|*Column / Attribute*
-|*Type of Data*
-|*Configuration*
+|@span{.center}{*Column / Attribute*}
+|@span{.center}{*Type of Data*}
+|@span{.center}{*Configuration*}
```
Comment (schanzer) [inline lessons/codap-bar-charts/langs/en-us/pages/plot-practice.adoc:20]: 🤮

File: lessons/codap-bar-charts/langs/en-us/pages/plot-practice.adoc [hunk 6]
```diff
@@ -77,11 +77,11 @@ The display tells us how many animals are male, hermaphrodite, and female.
...
 @n A bar chart showing the `species` of animals from the shelter.
-[cols="^1a,^1a,^1a", options="header"]
+[cols="1a,1a,1a", options="header"]
 |===
-|*Column / Attribute*
-|*Type of Data*
-|*Configuration*
+|@span{.center}{*Column / Attribute*}
+|@span{.center}{*Type of Data*}
+|@span{.center}{*Measure*}
```
Comment (schanzer) [inline lessons/codap-bar-charts/langs/en-us/pages/plot-practice.adoc:84]: 🤮

File: lessons/codap-exploring/langs/en-us/index.adoc [hunk 7]
```diff
@@ -122,7 +116,9 @@ Brainstorm other scenarios where Booleans are useful in and out of the programmi
 === Investigate
 [.lesson-instruction]
 --
-In pairs, complete @printable-exercise{pages/booleans.adoc}, making predictions about what a variety of Boolean expressions will return and testing them in the editor.
+- Turn to @printable-exercise{pages/booleans.adoc}.
+- Make a prediction about what a variety of Boolean expressions will return, and then test them in the @starter-file{boolean}.
```
Comment (schanzer) [inline lessons/codap-exploring/langs/en-us/index.adoc:120]: Honestly I'd suggest hitting kids over the head with it: "Write down your predictions _before_ testing them in CODAP". I know it's in the worksheet already, but in practice kids need to be reminded like crazy

File: lessons/codap-exploring/langs/en-us/pages/codap-exploration.adoc [hunk 8]
```diff
@@ -1,50 +1,94 @@
 = CODAP Exploration
```
Comment (schanzer) [inline lessons/codap-exploring/langs/en-us/pages/codap-exploration.adoc:1]: The changes to this worksheet are _excellent!_

**Review by schanzer — CHANGES_REQUESTED** [review]

File: lessons/codap-bar-charts/langs/en-us/pages/plot-practice-2.adoc (hunk 2 above)
Comment (retabak) [reply lessons/codap-bar-charts/langs/en-us/pages/plot-practice-2.adoc:20]: Sorry! I have mostly used @center for images. I think that's why it didn't occur to me. (I copied the @span thing from another place on the same page.)

File: lessons/codap-bar-charts/langs/en-us/pages/notes-plotting-and-displaying-data.adoc (hunk 1 above)
Comment (retabak) [reply lessons/codap-bar-charts/langs/en-us/pages/notes-plotting-and-displaying-data.adoc:18]: I was unaware of this! Good perspective, thanks. Will add.

Comment (retabak) [reply lessons/codap-bar-charts/langs/en-us/pages/notes-plotting-and-displaying-data.adoc:18]: Do you think we should add "dot plot" as vocab? I kinda don't? Noticing now how I bolded it up there.

File: lessons/codap-bar-charts/langs/en-us/pages/plot-practice.adoc (hunk 5 above)
Comment (retabak) [reply lessons/codap-bar-charts/langs/en-us/pages/plot-practice.adoc:20]: Lol.

File: lessons/codap-exploring/langs/en-us/index.adoc (hunk 7 above)
Comment (retabak) [reply lessons/codap-exploring/langs/en-us/index.adoc:120]: Good idea.

**Pull request closed without merging**